net-dns/dnssec-validator/files/dnssec-validator-2.2.3-ldflags.patch

```diff
@@ -0,0 +1,33 @@
+--- a/libsres/Makefile.in	2018-08-30 00:25:30.000000000 +0200
```

Comment:
Is this submitted upstream? Please add a link to the top.

Reply:
Nope, I patched this by myself. Should I report upstream?

Reply:
Yes.

Reply:
Done.